pkg/controller/operconfig/operconfig_controller.go

```diff
@@ -335,6 +335,17 @@ func (r *ReconcileOperConfig) Reconcile(ctx context.Context, request reconcile.R
 	allResources = append(allResources, deployments...)
 	r.podReconciler.SetResources(allResources)
 
+	// Remove ovnkube-node object if ovn db route doesn't exist in hypershift management cluster
+	if bootstrapResult.OVN.OVNKubernetesConfig.HypershiftConfig.Enabled && bootstrapResult.OVN.OVNKubernetesConfig.HypershiftConfig.Route == "" {
+		log.Printf("Removing ovnkube-node objects from rendered objs")
+		for i, obj := range objs {
+			if obj.GetName() == "ovnkube-node" && obj.GetKind() == "DaemonSet" {
+				objs = append(objs[:i], objs[i+1:]...)
+				r.status.SetDegraded(statusmanager.OperatorConfig, "ApplyOperatorConfig", "Waiting ovn db route to be created")
```

Comment:
Suggestion: add a `status.SetProgressing` function that does the same thing.

Reply:
added status.SetProgressing